src/Account/AccountManagement.tsx

```diff
@@ -124,101 +127,165 @@ class ManageAccountContainer extends React.Component<
+        </Flex>
+        <FastField
+          name={'email'}
+          label={CONSTANTS.EMAIL_LABEL}
+          value={fp.values.email}
+          component={FormikElements.Input}
+          disabled={mode === ManageAccountModes.EDIT}
+          required={true}
+        />
+        <ErrorMessage component={FormikElements.Error} name="email" />
+        <FastField
+          component={FormikElements.Input}
+          inputType={'password'}
+          label={CONSTANTS.PASSWORD_LABEL}
+          name={'password'}
+          required={mode === ManageAccountModes.CREATE}
+          value={fp.values.password}
+        />
+        <ErrorMessage component={FormikElements.Error} name="password" />
+        {mode === ManageAccountModes.EDIT ? (
+          <MaxWidthBox>
+            <FastField
+              label={CONSTANTS.NEW_PASSWORD_LABEL}
+              component={FormikElements.Input}
+              inputType={'password'}
+              name={'newPassword'}
+            />
+            <ErrorMessage component={FormikElements.Error} name="newPassword" />
+          </MaxWidthBox>
+        ) : (
+          ''
+        )}
+        <FastField
+          component={FormikElements.Select}
+          creatable={true}
+          isMulti={true}
+          label={CONSTANTS.DIETARY_RESTRICTIONS_LABEL}
+          name={'dietaryRestrictions'}
+          options={getOptionsFromEnum(DietaryRestriction)}
+          required={false}
+          value={fp.values.dietaryRestrictions}
+        />
+        <ErrorMessage
+          component={FormikElements.Error}
+          name="dietaryRestrictions"
+        />
+        <FastField
+          component={FormikElements.Select}
+          creatable={true}
+          label={CONSTANTS.PRONOUN_LABEL}
+          name={'pronoun'}
+          placeholder={CONSTANTS.PRONOUN_PLACEHOLDER}
+          options={getOptionsFromEnum(Pronouns)}
+          required={true}
+          value={fp.values.pronoun}
+        />
+        <ErrorMessage component={FormikElements.Error} name="pronoun" />
+        <FastField
+          component={FormikElements.Select}
+          label={CONSTANTS.SHIRT_SIZE_LABEL}
+          name={'shirtSize'}
+          options={getOptionsFromEnum(ShirtSize)}
+          required={true}
+          value={fp.values.shirtSize}
+        />
+        <ErrorMessage component={FormikElements.Error} name="shirtSize" />
+        <FastField
+          component={FormikElements.FormattedNumber}
+          label={CONSTANTS.PHONE_NUMBER_LABEL}
+          placeholder="+# (###) ###-####"
+          format="+# (###) ###-####"
+          name={'phoneNumber'}
+          required={true}
+          value={fp.values.phoneNumber}
+        />
+        <ErrorMessage component={FormikElements.Error} name="phoneNumber" />
+        <FastField
+          component={FormikElements.FormattedNumber}
+          label={CONSTANTS.BIRTH_DATE_LABEL}
+          placeholder="MM-DD-YYYY"
+          format="##-##-####"
+          name={'birthDate'}
```

Comment:
This guy seems to be broken (at least on edit).

When I try to submit an untouched edit form, the birth date breaks.
<img width="560" alt="image" src="https://user-images.githubusercontent.com/16010076/50405265-0f92d480-0780-11e9-81aa-4430fa5c4add.png">


When I try to edit it:
![screencast 2018-12-24 13-27-18](https://user-images.githubusercontent.com/16010076/50405255-c8a4df00-077f-11e9-81ef-0123b23bfe33.gif)

These are both regressions that used to work on `develop`.

Reply:
I'm unable to replicate this issue 😕

Reply:
Oh, I found the issue-- it has to do with the '0' before the 7.

Reply:
I believe the new fix should have fixed this